tests/kafkatest/tests/core/replica_scale_test.py

```diff
@@ -48,7 +46,7 @@ def teardown(self):
         self.zk.stop()
 
     @cluster(num_nodes=12)
-    @parametrize(topic_count=500, partition_count=34, replication_factor=3)
+    @parametrize(topic_count=100, partition_count=34, replication_factor=3)
```

Comment:
Since this PR is about upgrading to Python3 it probably shouldn't modify test parameters.

Reply:
Sorry, I decrease this variable to be able to run this tests in the Docker, otherwise it just freeze on my machine. Fixed.